database/seeds/DevSeeder.php

```diff
@@ -63,13 +63,27 @@ public function run() : void
             $job->job_applications()->saveMany(factory(JobApplication::class, 5))->create([
                 'job_poster_id' => $job->id
             ]);
+            //Then create one application with a priority user
+            $job->job_applications()->save(factory(JobApplication::class)->create([
+                'job_poster_id' => $job->id,
+                'applicant_id' => factory(Applicant::class)->create([
+                        'user_id' => factory(User::class)->state('priority')->create()->id
```

Comment:
Whenever a state is declared, it's `state()`, whenever a state is used, it's `states()` (thanks Laravel).

https://laravel.com/docs/5.8/database-testing#factory-states

Reply:
I'm not to sure about the syntax, Tryst helped me out with this haha. So what needs to be changed?